<?xml version="1.0" encoding="UTF-8"?>
<testrun name="DictionaryTest" project="HangToughGit" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="DictionaryTest" time="0.001">
    <testcase name="initializationError" classname="DictionaryTest" time="0.001">
      <error>java.lang.annotation.AnnotationFormatError: Duplicate annotation for class: interface org.junit.Test: @org.junit.Test(expected=class org.junit.Test$None, timeout=0)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations2(AnnotationParser.java:75)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations(AnnotationParser.java:52)
	at java.lang.reflect.Method.declaredAnnotations(Method.java:693)
	at java.lang.reflect.Method.getDeclaredAnnotations(Method.java:686)
	at java.lang.reflect.AccessibleObject.getAnnotations(AccessibleObject.java:175)
	at org.junit.runners.model.FrameworkMethod.getAnnotations(FrameworkMethod.java:146)
	at org.junit.runners.model.TestClass.addToAnnotationLists(TestClass.java:52)
	at org.junit.runners.model.TestClass.&lt;init&gt;(TestClass.java:42)
	at org.junit.runners.ParentRunner.&lt;init&gt;(ParentRunner.java:73)
	at org.junit.runners.BlockJUnit4ClassRunner.&lt;init&gt;(BlockJUnit4ClassRunner.java:55)
	at org.junit.internal.builders.JUnit4Builder.runnerForClass(JUnit4Builder.java:13)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.&lt;init&gt;(JUnit4TestReference.java:33)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestClassReference.&lt;init&gt;(JUnit4TestClassReference.java:25)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createTest(JUnit4TestLoader.java:48)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.loadTests(JUnit4TestLoader.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:452)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</error>
    </testcase>
  </testsuite>
</testrun>
